app/app/local.env

```diff
@@ -35,6 +35,10 @@ CHAT_URL=localhost
 GOOGLE_CLIENT_ID=
 GOOGLE_CLIENT_SECRET=
 
+# For Facebook integration (in profile's trust tab)
+FACEBOOK_CLIENT_ID=
```

Comment:
is this the same as App ID and an App Secret?

Reply:
Yes it is.